go.mod

```diff
@@ -124,4 +129,12 @@ require (
 )
 
 // replace with https://github.com/openshift/velero/tree/oadp-1.1
-replace github.com/vmware-tanzu/velero => github.com/openshift/velero v0.10.2-0.20230124191620-7c2f85743d78
+// TODO: revert to upstream
```

Comment:
Shouldn't this TODO be addressed in this PR ?

Reply:
Oops.. didn't see that's still there.
```suggestion
```